source/PreferencesPanel.cpp

```diff
@@ -692,7 +692,8 @@ void PreferencesPanel::DrawSettings()
 		"Interrupt fast-forward",
 		"Landing zoom",
 		SCROLL_SPEED,
-		DATE_FORMAT
+		DATE_FORMAT,
+		"Show parenthesis on Job Board"
```

Comment:
Trade prices aren't shown on the Job Board screen, they are part of the "Ports" map mode.

Also please add a tooltip.